packages/prisma/src/data/seedTEST.ts

```diff
@@ -506,6 +506,62 @@ async function seedTest(prisma: Prisma.PrismaClient) {
       update: {},
     })
   })
+
+  const goldMedalAchievement = await prisma.achievement.upsert({
+    where: { id: 11 },
+    create: {
+      id: 11,
+      name: 'Champion',
+      description: 'Du hast einen ersten Platz in einer Live-Session erreicht.',
+      icon: '/achievements/Champ.svg',
+      type: 'PARTICIPANT',
+      rewardedPoints: 200,
```

Comment:
add points/xp to the seedAchievements.ts script

Reply:
done